src/lib/y2network/sysconfig/config_writer.rb

```diff
@@ -67,6 +67,7 @@ def write(config, old_config = nil)
       def write_interface_changes(config, old_config)
         # Write ifroute files
         config.interfaces.each do |dev|
+          next if dev.name.empty?
```

Comment:
please document when this case can happen ( s390 not activated interface ?)

Reply:
Exactly, s390 interfaces that has not activated are added with an empty name and in that case it will crash.